src/main/java/AutoGarcon/DBUtil.java

```diff
@@ -354,6 +465,53 @@ public static boolean addUser(User user) {
         }
     }
 
+    /**
+     * saveOrder - saves the order and all of its order items to the database.   
+     * @param order - a full order containing all of its menu items
+     */
+    public static boolean saveOrder( Order order ){
+        Connection c = connectToDB(); 
+        CallableStatement stmt; 
+        ResultSet result;
+
+        try {
+            stmt = c.prepareCall( "{call CreateNewOrder(?, ?)}" ); 
+            stmt.setInt("tableID", order.getTableID() ); 
+            stmt.setInt("customerID", order.getCustomerID()); 
+            
+            //initialize order in DB and store the new orderID
+            result = stmt.executeQuery(); 
+            result.next();
+            int orderID = result.getInt("newOrderID"); 
+
+            order.setOrderID( orderID ); 
+
+            //add all orderitems to order
+            ArrayList<OrderItem> orderItems = order.getOrderItems();
+            for( OrderItem orderItem : orderItems ){
+                stmt = c.prepareCall( "{call AddItemToOrder(?, ?, ?, ?, ?)}" );
+                stmt.setInt("orderIDToAddTo", orderID);
+                stmt.setInt("menuItemIDToAdd", orderItem.getMenuItemID());
+                stmt.setInt("menuID", orderItem.getMenuID());
+                stmt.setInt("quantityToAdd", orderItem.getQuantity());
+                stmt.setString("commentsToAdd", orderItem.getComments());
+                stmt.executeQuery();
+            }
+
+            //complete order
+            stmt = c.prepareCall( "{call CompleteOrder(?)}" );
```

Comment:
Shouldn't complete order by used when the cook finishes the order to make the order as complete?